server.js

```diff
@@ -16,11 +16,90 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
+const { Schema } = mongoose;
+const PostSchema = new Schema({
+  text: {
+    type: String,
+    required: true,
+    minlength: 5,
+    maxlength: 140,
+    trim: true
+  },
+  hearts: {
+    type: Number,
+    default: 0
+  },
+  createdAt: {
+    type: Date,
+    default: Date.now
+  }
+})
+
+const Post = mongoose.model("Post", PostSchema)
+
 // Start defining your routes here
 app.get("/", (req, res) => {
   res.send("Hello Technigo!");
 });
 
+// This endpoint should return a maximum of 20 thoughts, sorted by createdAt to show the most recent thoughts first.
+app.get("/thoughts", async (req, res) => {
+  try {
+    const post = await Post.find().sort({createdAt: 'desc'}).limit(20).exec()
+    res.status(200).json({
+      success: true,
+      response: post,
+      message: "Thoughts retrieved!"
+    })
+  } catch (error) {
```

Comment:
I like that you type out 'error' instead of just an 'e', it doesn't hurt to be clear about what is referred to